docs/source/en/api/models/transformer2d.mdx

```diff
@@ -0,0 +1,29 @@
+# Transformer2D
+
+A Transformer model for image-like data. [`Transformer2DModel`] accepts discrete (classes of vector embeddings) or continuous (actual embeddings) inputs.
+
+When the input is **continous**:
+
+1. Project the input and reshape it to `(b, t, d)`.
+2. Apply standard Transformer action.
+3. Reshape to image.
+
+When the input is **discrete**:
+
+<Tip>
+
+It is assumed one of the input classes is the masked latent pixel. The predicted classes of the unnoised image don't contain a prediction for the masked pixel because the unnoised image cannot be masked.
+
+</Tip>
+
+1. Convert input (classes of latent pixels) to embeddings and apply positional embeddings.
+2. Apply standard Transformer action.
+3. Predict classes of unnoised image.
```

Comment:
Same as https://github.com/huggingface/diffusers/pull/3562/files#r1212553604 and https://github.com/huggingface/diffusers/pull/3562/files#r1212554581.